build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
     id 'com.avast.gradle.docker-compose' version "0.17.12" apply false
     id 'com.github.node-gradle.node' version '7.1.0' apply false
     id 'com.google.protobuf' version "0.9.5" apply false
-    id 'com.google.cloud.tools.jib' version '3.4.5' apply false
+    id 'com.google.cloud.tools.jib' version '3.5.2' apply false
```

Comment:
You'll get a merge conflict on this one - this has been moved (& thankfully as 3.5.2) to buildSrc